spaces/domains.json

```
@@ -858,5 +858,6 @@
   "proposals.powershift.properties": "powershiftproperties.eth",
   "gov.ohgeez.org": "levx.eth",
   "snapshot.panvala.com": "panvala.eth",
-  "vote.uwblockchain.ca": "uwblockchainclub.eth"
+  "vote.uwblockchain.ca": "uwblockchainclub.eth",
+  "vote.primapes.gg":"primapes.eth"
```

Please add a space between sub domain and ENS.
```suggestion
  "vote.primapes.gg": "primapes.eth"
```